orcid-model/src/main/java/org/orcid/jaxb/model/v3/release/record/Spam.java

```diff
@@ -0,0 +1,122 @@
+package org.orcid.jaxb.model.v3.release.record;
+
+import java.io.Serializable;
+
+import javax.xml.bind.annotation.XmlAccessType;
+import javax.xml.bind.annotation.XmlAccessorType;
+import javax.xml.bind.annotation.XmlElement;
+import javax.xml.bind.annotation.XmlRootElement;
+import javax.xml.bind.annotation.XmlType;
+
+import org.orcid.jaxb.model.v3.release.common.CreatedDate;
+import org.orcid.jaxb.model.v3.release.common.LastModifiedDate;
+import org.orcid.jaxb.model.v3.release.common.ReportedDate;
+
+import io.swagger.annotations.ApiModel;
+
+/**
+ * 
+ * @author Daniel Palafox
+ * 
+ */
+@XmlAccessorType(XmlAccessType.FIELD)
+@XmlType(propOrder = { "createdDate", "lastModifiedDate", "sourceType", "spamCounter", "reportedDate" })
+@XmlRootElement(name = "spam", namespace = "http://www.orcid.org/ns/spam")
+@ApiModel(value = "SpamV3_0")
```

Comment:
Since this is not going to be returned by the API, you don't need the XML annotation